handler/commentHandler_test.go

```diff
@@ -803,5 +805,57 @@ func Test_getCommandValue(t *testing.T) {
 			}
 		})
 	}
+}
 
+func Test_createIssueCommen(t *testing.T) {
+	desiredBody := "The content of the message with slack info"
+	tests := []struct {
+		title             string
+		message           []types.Message
+		wantedMessage     string
+		wantedErr         error
+		desiredGitHubBody *github.IssueComment
+	}{
+		{
+			title: "Desired message is found.",
+			message: []types.Message{
+				{Name: "slack", Value: "The content of the message with slack info"},
+				{Name: "docs", Value: "The content of the message with docs info"},
+			},
+			wantedMessage: "slack",
+			wantedErr:     nil,
+			desiredGitHubBody: &github.IssueComment{
+				Body: &desiredBody,
+			},
+		},
+		{
+			title: "User tried to set non existing message",
+			message: []types.Message{
+				{Name: "slack", Value: "The content of the message with slack info"},
+				{Name: "docs", Value: "The content of the message with docs info"},
+			},
+			wantedMessage:     "docs",
+			wantedErr:         fmt.Errorf("Unable to find message: docs"),
```

Comment:
I wonder whether we should test against a bool for wantedErr?  So false for the first scenario and true for the second.

It just means we don't have to chase the error output around if we choose to change that later.

Reply:
Good idea! Will keep in mind the approach 🍻